connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/CheckConnectJvmClientCompatibility.scala

```diff
@@ -95,7 +95,7 @@ object CheckConnectJvmClientCompatibility {
     } catch {
       case e: Throwable =>
         println(e.getMessage)
-        resultWriter.write(s"ERROR: ${e.getMessage}")
+        resultWriter.write(s"ERROR: ${e.getMessage}\n")
```

Comment:
This code seems to exist from Apache Spark 3.4.0 but SPARK-43977 is filed for 3.5.0 only. Do we need to backport this?

Reply:
@dongjoon-hyun Thanks for your review. Good question. Due to Spark 3.4.x will not add new modules for mima check with connect module, this issue should not be triggered, so I think just fixing it in Spark 3.5.0 is enough now

Reply:
Got it~